12-lambda/Dockerfile

```diff
@@ -8,18 +9,18 @@ ENV SHELL bash
 ENV AWS_SDK_VERSION 2.585.0
 ENV YARN_VERSION 1.22.15
 
+ARG TARGETARCH
+ARG AWSCLI_VERSION=2.9.15
+ADD --chmod=755 https://github.com/articulate/docker-consul-template-bootstrap/releases/latest/download/docker-consul-template-bootstrap_linux_${TARGETARCH} /entrypoint
+ADD --chmod=755 https://raw.githubusercontent.com/vishnubob/wait-for-it/81b1373f17855a4dc21156cfe1694c31d7d1792e/wait-for-it.sh /wait-for-it.sh
 COPY entrypoint-lambda.sh /
-ADD https://raw.githubusercontent.com/articulate/docker-consul-template-bootstrap/master/install.sh /tmp/consul_template_install.sh
-ADD https://raw.githubusercontent.com/articulate/docker-consul-template-bootstrap/master/wait-for-it.sh /wait-for-it.sh
 
 RUN npm install -g npm@7 \
     && npm install -g \
         aws-sdk@$AWS_SDK_VERSION \
         yarn@$YARN_VERSION \
-    && bash /tmp/consul_template_install.sh && rm /tmp/consul_template_install.sh \
-    && chmod a+rx /wait-for-it.sh \
     && groupadd $SERVICE_USER && useradd --create-home --home $SERVICE_ROOT --gid $SERVICE_USER --shell /bin/bash $SERVICE_USER
 
 WORKDIR $SERVICE_ROOT
 
-ENTRYPOINT ["/entrypoint-lambda.sh"]
+ENTRYPOINT [ "/entrypoint-lambda.sh", "/entrypoint" ]
```

Comment:
I honestly don't know what made /entrypoint-lambda.sh unique, so mainly just getting your second peak on this line and 👍.

Reply:
The lambda entrypoint links the aws-sdk in npm and then runs the given command. I'm not exactly sure why it needs to be linked, but there's no harm in it